src/i18n/en.ts

```diff
@@ -142,8 +142,10 @@ export default {
 		"ROW": "row"
 	},
 	"TABLE_TOOLBAR": {
-		"ACTION_BAR": "Table Action Bar",
-		"BATCH_TEXT": "items selected",
+		"ACTION_BAR": "Table action bar",
+		"BATCH_TEXT": "",
```

Comment:
do we still need this?

Reply:
@theiliad yes it is needed for the `_batchTextLegacy: Overridable = this.i18n.getOverridable("TABLE_TOOLBAR.BATCH_TEXT")` call for compatibility reason (in case someone customized that message in their project).

We have to leave an empty string in en.ts like this, otherwise `i18n.getValueFromPath` will thrown an exception when `i18n.getOverridable("TABLE_TOOLBAR.BATCH_TEXT")` is called.